### ISSUE TITLE
Community pinboard

### ISSUE THREAD

**Issue by Andersson007** [ansible-collections/community.general#3273]
### Summary

This is a pinboard for general discussions and announcements

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by mator** [comment]
@Andersson007 , @felixfontein ,
#3242 could be probably used as a sample (simple to fix, just create a PR) to introduce new ansible developers...

**Comment by Andersson007** [comment]
@mator yes, thanks! there's the [hackmd.io file](https://hackmd.io/CqUQNTEqRBCHBTC4WF-9Aw?both), feel free to update.
Also would be nice to mark the issue somehow to prevent fixing it before the hackathon:)

**Comment by mator** [comment]
> Also would be nice to mark the issue somehow to prevent fixing it before the hackathon:)

i believe another label like 'hackathon21' would make it easier to search =)

**Comment by Andersson007** [comment]
> > Also would be nice to mark the issue somehow to prevent fixing it before the hackathon:)
> 
> i believe another label like `hackathon21` would make it easier to search =)

sounds good, and maybe put explicitly "Please don't fix this before the hackathon 09.21 <reg link>"

**Comment by dericcrago** [comment]
[AnsibleFest]: https://www.ansible.com/ansiblefest
[Ansible Contributor Summit]: https://ansiblecs202109.eventbrite.com/?aff=github
[Hackathon]: https://hackmd.io/@ansible-community/contrib-summit-202109#Hackathon
[Ansible Contributor Summit 2021.09 HackMD]: https://hackmd.io/@ansible-community/contrib-summit-202109

**Reminder - [AnsibleFest] and [Ansible Contributor Summit] are 1 week away!**

In case you missed it, we will also be having a **[Hackathon]** throughout the entire **[Ansible Contributor Summit]** and **[AnsibleFest]**. This is a great opportunity to collaborate in real time with other members of the Ansible Community!

For more info and the latest updates, please see the **[Ansible Contributor Summit 2021.09 HackMD]**.

**Comment by Andersson007** [comment]
[Ansible Contributor Summit]: https://ansiblecs202109.eventbrite.com/?aff=github
[Hackathon]: https://hackmd.io/@ansible-community/contrib-summit-202109#Hackathon
[Ansible Contributor Summit 2021.09 HackMD]: https://hackmd.io/@ansible-community/contrib-summit-202109

**Reminder** - Day 1 of the [Ansible Contributor Summit] and the [Hackathon] start tomorrow (Tuesday, 28 September 2021)!

For more info and the latest updates, please see the **[Ansible Contributor Summit 2021.09 HackMD]**.

**Comment by Andersson007** [comment]
[Ansible Contributor Summit]: https://ansiblecs202109.eventbrite.com/?aff=github
[Hackathon]: https://hackmd.io/@ansible-community/contrib-summit-202109#Hackathon
[Ansible Contributor Summit 2021.09 HackMD]: https://hackmd.io/@ansible-community/contrib-summit-202109

**Reminder** - Day 2 of the [Ansible Contributor Summit] and the last day of the [Hackathon] start tomorrow (Friday, 1 October 2021)!

For more info and the latest updates, please see the **[Ansible Contributor Summit 2021.09 HackMD]**.

**Comment by v1v** [comment]
I don't know whether this is the right place to ask, I saw the `general discussions` topic.

Is there any publicly available telemetry data that I could query to know roughly how many installations per version have been downloaded for this collection? Or even how many times a plugin has been used in this collection? I'd like to know the adoption to keep working/adding features in some of the plugins.

Thanks

**Comment by felixfontein** [comment]
@v1v ansible does not collect or report any telemetry. There is only a global collection download counter on [Ansible Galaxy](https://galaxy.ansible.com/community/general), but that isn't really helpful since a lot of other collections install community.general in CI (while yet other users simply install the `ansible` community package and also obtain a copy of the collection that way).

**Comment by v1v** [comment]
Thanks @felixfontein, I guess, either I build something to collect those metrics and enable users to opt-out or do nothing ad keep adding features :)

If I add this telemetry to the plugins I'm interested, is there anything I should be careful about? Just in case a similar question was raised in the past.

Thanks again

**Comment by mator** [comment]
@v1v why would you like to collect telemetry, and how you would do it? What does it mean "opt-out", can I be free and not "opt-in" in the first place? Thanks

**Comment by felixfontein** [comment]
I don't think we will ever have something that's opt-out. I'm also not sure whether opt-in would be accepted.

**Comment by v1v** [comment]
This is related to adding telemetry data to one of the plugins

> why would you like to collect telemetry, and how you would do it? 

Ideally, to help me to understand if it's used and keep contributing adding new features. Sending a quite basic data with:
- collection name
- plugin name
- ansible version

To an endpoint.

> What does it mean "opt-out", can I be free and not "opt-in" in the first place? 

Sorry for the misleading, I meant that the telemetry data would be sent by default, unless a property is configured to disable it.

For instance, some other FOSS projects use a similar concept -> https://github.com/jenkinsci/jep/blob/master/jep/214/README.adoc . Though, it's provided by the core itself.

**Comment by russoz** [comment]
Hi @felixfontein @Andersson007 I think this one has also been superseded by the Ansible Forum. Last post in 2022, may I suggest we close it?

**Comment by felixfontein** [comment]
I agree. Everyone who's still subscribed: if you haven't, please join the [Ansible forum](https://forum.ansible.com/) :)
